perf/lua/column_insert.lua

```diff
@@ -18,6 +18,10 @@ local benchmark = require('benchmark')
 local USAGE = [[
    engine <string, 'memtx'>          - space engine to use for the test
    wal_mode <string, 'write'>        - write-ahead log mode to use for the test
+   sparse_mode <string, 'rand'>      -
+     * seq - the first 10% (column_count_batch / column_count_total) columns are
+             filled in sequential order;
+     * rand - sparse columns are randomly distributed;
```

Comment:
Let's add details:
- in case of non batch mode columns are randomly chosen for every row
- in case of batch mode columns are randomly chosen for every batch

Reply:
Added:
```diff
--- a/perf/lua/column_insert.lua
+++ b/perf/lua/column_insert.lua
@@ -21,7 +21,9 @@ local USAGE = [[
    sparse_mode <string, 'rand'>      -
      * seq - the first 10% (column_count_batch / column_count_total) columns are
              filled in sequential order;
-     * rand - sparse columns are randomly distributed;
+     * rand - sparse columns are randomly distributed:
+              in non-Arrow mode columns are randomly chosen for every row;
+              in Arrow mode columns are randomly chosen for every batch.
    column_count_total <number, 1000> - number of columns in the test space
    column_count_batch <number, 100>  - number of columns in the record batch
    row_count_total <number, 1000000> - number of inserted rows
```